src/acl/group.ts

```diff
@@ -0,0 +1,212 @@
+import { acl } from "../constants";
+
+/**
+ * Find out what Access Modes have been granted to a given Group of agents specifically for a given Resource.
+ *
+ * Keep in mind that this function will not tell you what access the given Group has through other ACL rules, e.g. public permissions.
+ *
+ * Also, please note that this function is still experimental: its API can change in non-major releases.
+ *
+ * @param resourceInfo Information about the Resource to which the given Group may have been granted access.
+ * @param group URL of the Group for which to retrieve what access it has to the Resource.
+ * @returns Which Access Modes have been granted to the Group specifically for the given Resource, or `null` if it could not be determined (e.g. because the current user does not have Control Access to a given Resource or its Container).
+ */
+export function unstable_getGroupAccessOne(
+  resourceInfo: unstable_WithAcl & WithResourceInfo,
+  group: UrlString
+): unstable_Access | null {
+  if (unstable_hasResourceAcl(resourceInfo)) {
+    return unstable_getGroupResourceAccessOne(
+      resourceInfo.acl.resourceAcl,
+      group
+    );
+  }
+  if (unstable_hasFallbackAcl(resourceInfo)) {
+    return unstable_getGroupDefaultAccessOne(
+      resourceInfo.acl.fallbackAcl,
+      group
+    );
+  }
+  return null;
+}
+
+/**
+ * Find out what Access Modes have been granted to specific Groups of agents for a given Resource.
+ *
+ * Keep in mind that this function will not tell you what access each Group has through other ACL rules, e.g. public permissions.
+ *
+ * Also, please note that this function is still experimental: its API can change in non-major releases.
+ *
+ * @param resourceInfo Information about the Resource to which the given Group may have been granted access.
+ * @returns Which Access Modes have been granted to which Groups specifically for the given Resource, or `null` if it could not be determined (e.g. because the current user does not have Control Access to a given Resource or its Container).
+ */
+export function unstable_getGroupAccessAll(
+  resourceInfo: unstable_WithAcl & WithResourceInfo
+): unstable_AgentAccess | null {
+  if (unstable_hasResourceAcl(resourceInfo)) {
+    const resourceAcl = unstable_getResourceAcl(resourceInfo);
+    return unstable_getGroupResourceAccessAll(resourceAcl);
+  }
+  if (unstable_hasFallbackAcl(resourceInfo)) {
+    const fallbackAcl = unstable_getFallbackAcl(resourceInfo);
+    return unstable_getGroupDefaultAccessAll(fallbackAcl);
+  }
+  return null;
+}
+
+/**
+ * Given an ACL LitDataset, find out which access modes it provides to a Group for its associated Resource.
+ *
+ * Keep in mind that this function will not tell you:
+ * - what access the given Group has through other ACL rules, e.g. public permissions.
+ * - what access the given Group has to child Resources, in case the associated Resource is a Container (see [[unstable_getGroupDefaultAccessModesOne]] for that).
```

Comment:
```suggestion
 * - what access members of the given Group have through other ACL rules, e.g. public permissions.
 * - what access members of the given Group have to child Resources, in case the associated Resource is a Container (see [[unstable_getGroupDefaultAccessModesOne]] for that).
```